database/queries/index.js

```diff
@@ -1,3 +1,3 @@
-module.exports.getUserByGoogleID = require('./getUserByGoogleID');
-module.exports.getUserByID = require('./getUserByID');
-module.exports.insertUser = require('./insertUser');
\ No newline at end of file
+module.exports.getUserByGoogleID = require('./get-user-by-google-id');
+module.exports.getUserByID = require('./get-user-by-id');
+module.exports.insertUser = require('./insert-user');
```

Comment:
Renaming to make naming convention consistent. Unrelated to main content of pull request.